sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -56,11 +57,13 @@
   public static final String SONAR_PREFIX = "sonar.";
   public static final String DEFAULTS_PREFIX = "defaults";
   public static final String PLUGINS = "plugins";
+  public static final String SCANNERS = "scanners";
   private static final String PUBLIC_VERSIONS = "publicVersions";
   private static final String PRIVATE_VERSIONS = "privateVersions";
   private static final String ARCHIVED_VERSIONS = "archivedVersions";
   private static final String DEV_VERSION = "devVersion";
   private static final String LATEST_KEYWORD = "LATEST";
+  private static final String FLAVORS_PREFIX = "flavors";
```

Comment:
NP: I would flip these 2 lines, so we group related Striings. "flavors" is a property, "LATEST" is a value.